src/spyglass/spikesorting/v0/spikesorting_artifact.py

```diff
@@ -36,19 +55,28 @@ class ArtifactDetectionParameters(SpyglassMixin, dj.Manual):
     """
 
     def insert_default(self):
-        """Insert the default artifact parameters with an appropriate parameter dict."""
-        artifact_params = {}
-        artifact_params["zscore_thresh"] = None  # must be None or >= 0
-        artifact_params["amplitude_thresh"] = 3000  # must be None or >= 0
-        # all electrodes of sort group
-        artifact_params["proportion_above_thresh"] = 1.0
-        artifact_params["removal_window_ms"] = 1.0  # in milliseconds
-        self.insert1(["default", artifact_params], skip_duplicates=True)
-
-        artifact_params_none = {}
-        artifact_params_none["zscore_thresh"] = None
-        artifact_params_none["amplitude_thresh"] = None
-        self.insert1(["none", artifact_params_none], skip_duplicates=True)
+        """Insert the default artifact parameters."""
```

Comment:
Might be more readable to declare both dictionaries first and then add them to the insert call e.g.:
```python
artifact_params_default = {
        "zscore_thresh": 5.0,  # must be None or >= 0
        "amplitude_thresh": 3000,  # must be None or >= 0
        "proportion_above_thresh": 0.5,  # must be > 0 and <= 1
        "removal_window_ms": 1.0,  # in milliseconds
}
artifact_params_none = {
        "zscore_thresh": None,  # must be None or >= 0
        "amplitude_thresh": None,  # must be None or >= 0
}
self.insert(
    [("default", artifact_params_default),
     ("none", aritifact_params_none)
    ],
skip_duplicates=True)
```